### PR TITLE
create admin view infrastucture

### PR DESCRIPTION
**Pull request by RaphaelBS-WCS**

### CODE REVIEW

**Review by Vannou28 — DISMISSED** [review]

File: src/View/Admin/Home/Component/_admin_navbar.html.twig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        </a>
+        <a class="navbar-brand-subtitle" href="/">
+            VOYANCE - MEDIUM - TAROT    
+        </a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="far fa-eye"></span>
+        </button>
+        <div class="collapse navbar-collapse justify-content-end" id="navbarNavAltMarkup">
+            <div class="navbar-nav">
+                <a class="nav-link" href="http://localhost:8000/">ACCUEIL VISITOR</a>
```
Comment (LucyMerlier) [inline src/View/Admin/Home/Component/_admin_navbar.html.twig:14]: routes only start at the first "/", if you want this link to go to your home page, href should just be "/"
+ "VISITOR" is not french ;)

File: src/View/Admin/Home/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{% extends '/Admin/admin_layout.html.twig' %}
+
+{% block stylesheet %}
+    {{ parent() }}
+{% endblock %}
```
Comment (LucyMerlier) [inline src/View/Admin/Home/index.html.twig:5]: if you want what's in the parent block and nothing more, just don't put a block stylesheet here

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/Admin/Home/Component/_admin_navbar.html.twig (hunk 1 above)
Comment (RaphaelBS-WCS) [reply src/View/Admin/Home/Component/_admin_navbar.html.twig:14]: Oh yes, thank you

File: src/View/Admin/Home/index.html.twig (hunk 2 above)
Comment (RaphaelBS-WCS) [reply src/View/Admin/Home/index.html.twig:5]: well, things are going to come next

**Review by Vannou28 — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**